Doc/whatsnew/3.7.rst

```diff
@@ -158,6 +158,11 @@ API and Feature Removals
   Python 3.1, and has now been removed.  Use the :func:`~os.path.splitdrive`
   function instead.
 
+* Deprecated :meth:`format` from :mod:`locale`, use the :meth:`format_string`
+  instead. Added another argument *monetary* in :meth:`format_string` of
```

Comment:
format hasn't been removed, so the deprecation should go in the deprecation section.  The feature addition of the monetary keyword should go in the 'improved modules' section.